examples/cpp/pyperf/PyPerfProfiler.cc

```diff
@@ -256,10 +262,8 @@ bool PyPerfProfiler::populatePidTable() {
       continue;
     }
     pid_config_map.update_value(pid, pidData);
-
```

Comment:
Can you please undo unrelated changes? (blank additions/removals in unrelated code)

Reply:
👍

Reply:
(was not pushed yet, leaving this unresolved)

Reply:
@marcin-ol was not pushed yet' let's remove all unrelated formatting fixes in this PR, blank changes etc.

Reply:
Should be cleaned up now.